.github/workflows/verify-server-start.yml

```
@@ -26,13 +26,23 @@
 
       - name: Install Dependencies
         run: npm ci
+      
+      - name: Build Project
+        run: npm run build
+
+      - name: Verify Server Startup (no cluster)
+        working-directory: ./dist  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)